package.json

```diff
@@ -3,13 +3,14 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "firebase": "^8.3.3",
+    "firebase": "^8.4.1",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
     "react-firebase-hooks": "^3.0.3",
     "react-router": "^5.2.0",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "4.0.1"
+    "react-scripts": "4.0.1",
+    "uuid": "^8.3.2"
```

Comment:
Please could you remove ```"uuid": "^8.3.2"``` from your package.json? We’re not using it.

Reply:
Yess, I agree I forgot to remove it